test/system-test.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const webpack = require('webpack');
+const Plugin = require('../index.js');
+const path = require('path');
+const assert = require('assert');
+const fs = require('fs');
+
+describe('Smoke test for the executable script', function() {
+    beforeEach(() => {
+        fs.writeFileSync('test/test-data.js', 'const a = __(\'test.key\');');
+    });
+
+    afterEach(() => {
+        fs.unlinkSync('test/test-data.js');
```

Comment:
@vigneshshanmugam  I thought to use rimraf. But just because it was an additional package, i was a little lazy to do that ;)